### PR TITLE
[PLU-102] feat(rich-text): apply tiptap editor for string and multiline fields

### PR DESCRIPTION
**Pull request by stanleynguyen**
## Problem

Apply the new tiptap rich text editor to `string` and `multiline` fields as well

## Solution

- Hide rich formatting bar
- Output as text instead of HTML

## Tests

- [x] old fields are rendering exactly the same as before and can be edited and triggered correctly
- [x] new pipelines can be edited and trigger to the intended result

## Deploy Notes

- [x] new editor has been deployed for new `rich-text` fields (i.e. postman email body)

### CODE REVIEW

**Comment by ogp-weeloong** [comment]
can rebase this X_X

**Review by ogp-weeloong — APPROVED** [review]
Nice, LGTM

Tested
- Backward compat; renders old fields correctly (using telegram action)
- Able to insert vars into new fields

File: packages/frontend/src/components/RichTextEditor/StepVariablePlugin.ts [hunk 1]
```diff
@@ -92,7 +92,7 @@ export const StepVariable = Node.create<VariableOptions>({
     ]
   },
   renderText({ node }) {
-    return node.attrs.id
+    return `{{${node.attrs.id}}}`
```
Comment (m0nggh) [inline packages/frontend/src/components/RichTextEditor/StepVariablePlugin.ts:95]: ps @stanleynguyen can i ask when does this occur ah

Comment (stanleynguyen) [reply packages/frontend/src/components/RichTextEditor/StepVariablePlugin.ts:95]: This is when we do `editor.getText()`

**Pull request merged**